app/models/user.rb

```diff
@@ -350,11 +350,13 @@ def automator?(track = nil)
     end
   end
 
-  def trainer?(_track = nil)
+  def trainer?(track = nil)
     return true if staff?
-    return true if maintainer?
-    return true if supermentor?
+    return reputation_for_track(track) >= MIN_TRACK_REP_FOR_TRAINER if track.present?
 
     false
```

Comment:
I think rather than false here, this need to be "any track.rep > MIN_TRACK_REP_FOR_TRAINER`

Reply:
I'm not sure how to do that though:)